### ISSUE TITLE
SQLWatchDashboard - Duplicate value Dim Database - column database_name

### ISSUE THREAD

**Issue by LeosHorna** [marcingminski/sqlwatch#349]
Hello, I have 4 SQL servers. After running SQL Import, the SQLWatchDashboard report generates an error:

The database_name column in the Dim Database table contains a duplicate tempdb value, which is not allowed for columns on one side of an N: 1 session or for columns used as the table's primary key.

Version SQLWatch installed from: SQLWATCH.4.1.0.28258.20210510154229.zip
but version in table sqlwatch_app_version : 4.1.0.24109

Older version 3 worked fine.

Thank you in advance for the solution, Leoš

**Comment by marcingminski** [comment]
can you show me output from `SELECT * FROM [dbo].[sqlwatch_meta_database]` please?

**Comment by LeosHorna** [comment]
> can you show me output from `SELECT * FROM [dbo].[sqlwatch_meta_database]` please?

[Select-210512a.xlsx](https://github.com/marcingminski/sqlwatch/files/6466445/Select-210512a.xlsx)

**Comment by marcingminski** [comment]
This looks ok to me. Are you using the PBI dashboard from the 4.1 release? can you try one from 4.0 or 3.0?

**Comment by LeosHorna** [comment]
Yes, from 4.1 (SQLWATCH Dashboard 20210430.pbix) , after apply changes in PowerBI Desktop:
![Error-ApplyQueryChange-210512a](https://user-images.githubusercontent.com/32358964/117985343-e6674480-b338-11eb-93a9-081fac3c6ff7.PNG)

After a clean installation of SQL Watch 4.0 on two servers and running SQLImport, the report shows the same error.

Version 3.0.9 on other servers is fine.

**Comment by marcingminski** [comment]
This seems to be caused by a relation between Blockers and Dim database. This relation is on the database_name column rather than pbi_database_id. Feels wrong.
![image](https://user-images.githubusercontent.com/8288333/118329991-0ba5af80-b4ff-11eb-8998-1e4fd8b9b965.png)

**Comment by marcingminski** [comment]
If we change the relation to many to many it will allow the dashboard to load but the blockers tab may not show correct data as it will not be able to link database with the correct server
![image](https://user-images.githubusercontent.com/8288333/118330148-43145c00-b4ff-11eb-8c56-01f6ec7ae6c5.png)

**Comment by marcingminski** [comment]
Although, thinking about it. Seeing there is a relation between the Dim Sever too, the Dim Database will be filtered by the Dim Server so the only available databases in Dim Database will have already been filtered by Dim Server. So the Blockers may actually work. I will have to test this.

**Comment by LeosHorna** [comment]
Hello, I've been gone a few days.
Thank you for the advice.
I deleted relation and deleted the Blocking page.
This suits us.

Thank you so far, Leoš
